generator.yaml

```diff
@@ -28,8 +28,14 @@ resources:
           input_fields:
             PolicyName: Name
     hooks:
+      sdk_read_one_post_set_output:
+        template_path: hooks/policy/sdk_read_one_post_set_output.go.tpl
       sdk_create_post_set_output:
         template_path: hooks/policy/sdk_create_post_set_output.go.tpl
+      sdk_update_post_set_output:
+        template_path: hooks/policy/sdk_update_post_set_output.go.tpl
+    update_operation:
+      custom_method_name: customUpdatePolicy
```

Comment:
I just noticed something... I see you've added a `custom_method_name` configuration option to the `generator.yaml` file for the Policy resource (unlike the Role resource, which uses the `sdk_update_post_set_output` generic hook point.

I prefer to use the generic hook points instead of creating a custom update method. Can you change that?

Reply:
> Never mind, @RedbackThomson, I realized now that you had to use a custom update method because there isn't an actual UpdatePolicy API call...

Yeah this is a non-obvious configuration problem you have when trying to use hooks. I'll remove the hook point, though. Good call.